HW_3/task_4.py

```diff
@@ -0,0 +1,25 @@
+"""
+Написать программу по переводу целого числа из десятичной системы счисления в двоичную.
+
+Ввод: значение типа <int>
+Вывод: значение типа <int>
+
+Примеры:
+45
+101101
+
+3
+11
+
+2
+10
+"""
+num = int(input('Введите число '))
+
+binar = '' # строковая переменная 
+while num > 0: 
+    binar_elem = num %2 # от деления целого числа на 2 получаем остаток. Это последний символ бинарного числа
+    num = num//2 # тут берем целую часть от деления на 2
+    binar+=str(binar_elem)
+print(int(binar[::-1])) # :: – это разделитель аргументов среза. 
```

Comment:
Пожалуйста, не забывайте про PEP8.
```suggestion
while num: 
     binar += str(num % 2)  # от деления целого числа на 2 получаем остаток. Это последний символ бинарного числа
    num //= 2 # тут берем целую часть от деления на 2
print(int(binar[::-1])) # :: – это разделитель аргументов среза. 
```

Reply:
PEP8 стоит у меня. Но он очень плохо работает. Так же как и у Натальи на втором семинаре.